src/utils/__tests__/adjust-comments-on-sorted-nodes.spec.ts

```diff
@@ -98,11 +98,16 @@ test('it does not affect comments after all import declarations', () => {
     expect(importNodes).toHaveLength(3);
     const finalNodes = [importNodes[2], importNodes[1], importNodes[0]];
     const adjustedNodes = adjustCommentsOnSortedNodes(importNodes, finalNodes);
-    expect(adjustedNodes).toHaveLength(3);
-    expect(leadingComments(adjustedNodes[0])).toEqual([]);
-    expect(trailingComments(adjustedNodes[0])).toEqual([]);
+    expect(adjustedNodes).toHaveLength(4);
     expect(leadingComments(adjustedNodes[1])).toEqual([]);
-    expect(trailingComments(adjustedNodes[1])).toEqual([]);
+    // "final 1" is attached as a trailing-comment for import from "a"
+    // but "final 2" is detached so it stays with the bottom-of-imports
```

Comment:
This behavior seems a bit odd to me.  The test description is that comments after all the imports should not be moved, and I think I agree with that. The only kind of "trailing" comment that feels like it should move with an import is one that is on the same line.  Comments that start on the line below an import feel like they shouldn't be associated with that import.

Reply:
I could accept that argument.

If you want to change that behavior, it’s just removing the “+ 1” on the `trailingGap` comparison (plus updating Readme and test-cases)

Reply:
I'm implementing this change now, and then updating the comments/snapshots to match!

Reply:
[Done ✅ ]